data/mods/steam/engines.json

```diff
@@ -0,0 +1,160 @@
+[
+  {
+    "abstract": "engine_steam",
+    "type": "ENGINE",
+    "category": "veh_parts",
+    "name": "external combustion engine",
+    "material": "iron",
+    "symbol": ":",
+    "color": "light_cyan"
+  },{
+    "id": "i2_steam",
+    "copy-from": "engine_steam",
+    "type": "ENGINE",
+    "name": "i2 steam engine",
+    "description": "A high power 2 piston steam engine",
+    "weight": 20000,
+    "volume": 25,
+    "price": 25000,
+    "displacement": 200
+  },{
+    "id": "steam_turbine",
+    "copy-from": "engine_steam",
+    "type": "ENGINE",
+    "name": "steam turbine steam engine",
+    "description": "A high power 2 piston steam engine",
+    "weight": 20000,
+    "volume": 25,
+    "price": 25000,
+    "displacement": 500
+  },{
+      "id":"steam",
+      "type":"AMMO",
+      "name": "steam",
+      "name_plural": "steam",
+      "description": "Hot water, very hot water",
+      "category": "fuel",
+      "weight": 0.4,
+      "volume": 1,
+      "price": 40,
+      "price_postapoc": 800,
+      "count": 250,
+      "stack_size": 250,
+      "phase": "liquid",
```

Comment:
Technically gas. This means that you should flag it as never dropping. I think "NO_DROP" flag would do the job.